Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/README.md

```diff
@@ -0,0 +1,238 @@
+
+[^1]: Knill et al (2008 Phys. Rev. A 77 012307)
+[^2]: C A Ryan et al 2009 New J. Phys. 11 013034
+[^3]: X. Xue et al Phys. Rev. X 9, 021011
+[^4]: A. D. Córcoles et al Phys. Rev. A 87, 030301(R)
+
+## Experimental Setup
+<img align="right" width="400" src="https://github.com/maximilianqm/max-dev/blob/main/Quantum-Control-Applications/Superconducting/Multiple%20Flux%20Tunable%20Transmons/Use%20Case%202%20-%20Two%20Qubit%20Randomized%20Benchmarking/setup.png">
+
+The use-case in this example is tailored for a superconducting quantum processor using flux-tunable transmon qubits, where we focus on a subset of two qubits that are capacitively coupled to each other. Single qubit operations are controlled by sending microwave pulses through a xy-line that is capacitively coupled to the individual qubits. The two-qubit gate is implemented by a controlled-Z (CZ) gate utilizing the fast-flux lines to rapidly change the qubit frequencies and the capacitive coupling between both qubits. Part of the optimization protocol for tuning up a CZ gate can be found in the use-case [*Two-Qubit Gate Optimization*](https://github.com/qua-platform/qua-libs/tree/2qb-RB-usecase/Quantum-Control-Applications/Superconducting/Multiple%20Flux%20Tunable%20Transmons/Use%20Case%201%20-%20Two%20qubit%20gate%20optimization%20with%20cryoscope).
+
+## Prerequisites
+- Calibrated Single Qubit Gates
+- Calibrated CZ Gate
+- Calibrated Measurement Protocol for Qubit State Discrimination
+
+# Quick User Guide
+For a quick implementation just clone the [*qua-libs repository*](https://github.com/qua-platform/qua-libs/) or download the [*Use Case 2 - Two Qubit Randomized Benchmarking*](https://github.com/qua-platform/qua-libs/tree/2qb-RB-usecase/Quantum-Control-Applications/Superconducting/Multiple%20Flux%20Tunable%20Transmons/Use%20Case%202%20-%20Two%20Qubit%20Randomized%20Benchmarking) folder and edit the [*two_qubit_rb_example.py*](https://github.com/qua-platform/qua-libs/blob/2qb-RB-usecase/Quantum-Control-Applications/Superconducting/Multiple%20Flux%20Tunable%20Transmons/Use%20Case%202%20-%20Two%20Qubit%20Randomized%20Benchmarking/two_qubit_rb_example.py) python file.
+
+## Single Qubit Gates
+The function for the single qubit gates requires that the operation "x" was previously calibrated by the user and corresponds to a pi-pulse on the target qubits. The *amp=x* condition inside the *baker.play* statement allows to scale the amplitude of the pulse. Together with the first *baker.frame_rotation_2pi* it allows the *baker.play* statement to act as X and Y gates by shifting the frame of the control signal, thus realizing rotations around the x- and y-axis. The second *baker.frame_rotation_2pi* resets the frame and additionally allows for rotations around the z-axis.
+```python
+def bake_phased_xz(baker: Baking, q, x, z, a):
+    element = f"qubit{q}_xy"
+    baker.frame_rotation_2pi(-a, element)
+    baker.play("x", element, amp=x)
```

Comment:
Can the X gate be shorter than 40ns?

Reply:
Also, the docstring of baker.play() says that 'amp' is the amplitude of the pulse, is it the absolute amplitude or a prefactor?

Reply:
prefactor, just like the amp in QUA

Reply:
the minimal time here is 40nsec.... it's based on the for-switch construction, with operations occuring on multiple elements simultaneously split across the for-switch